src/contracts/interfaces/IRwaAToken.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title IRwaAToken
+ * @author Aave
+ * @notice Defines the basic interface for an RwaAToken.
+ */
+interface IRwaAToken {
+  /**
+   * @notice ERC-20 Permits are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function permit(
+    address owner,
+    address spender,
+    uint256 value,
+    uint256 deadline,
+    uint8 v,
+    bytes32 r,
+    bytes32 s
+  ) external;
+
+  /**
+   * @notice ERC-20 Approvals are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function approve(address spender, uint256 amount) external returns (bool);
+
+  /**
+   * @notice ERC-20 Allowances are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function increaseAllowance(address spender, uint256 addedValue) external returns (bool);
+
+  /**
+   * @notice ERC-20 Allowances are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function decreaseAllowance(address spender, uint256 subtractedValue) external returns (bool);
+
+  /**
+   * @notice ERC-20 Transfers are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function transfer(address recipient, uint256 amount) external returns (bool);
+
+  /**
+   * @notice ERC-20 Transfers are not supported for RWA aTokens.
+   * @dev Reverts if called.
+   */
+  function transferFrom(address sender, address recipient, uint256 amount) external returns (bool);
+
+  /**
+   * @notice RWA aToken transfers are not supported in liquidations.
```

Comment:
```suggestion
   * @notice Transfers are not supported in liquidations.
```
Just use `Transfers` in all functions' docs, making it general (not `ERC-20 transfers` or `RWA aToken transfers`)